int32 num_boxes#the number of boxes
float64[] boxes_xywh  # xywh format
float64[] boxes_cls   #categorys
float64[] boxes_conf  # confidence

